bin/setup

```diff
@@ -20,7 +20,7 @@ FileUtils.chdir APP_ROOT do
   system! "bundle config --local ignore_messages.rpush true"
   system("bundle check") || system!("bundle install")
 
-  system! "gem list \"^foreman$\" -v 0.87.1 -i --silent || gem install foreman"
+  system! "gem list \"^foreman$\" -v 0.87.2 -i --silent || gem install foreman"
```

Comment:
I'll just reiterate Jacob's doubt that this version test is needed here to stir the pot - I'm totally okay with testing that we get a 0 exit status for _any_ version of foreman, or periodically updating this as needed.

Since it's not in the gemfile, dependabot won't remember to do this for us.

Reply:
I agree with this, I think we can let go of the hard version constraint.